horovod/common/operations.cc

```diff
@@ -1109,6 +1158,27 @@ void BackgroundThreadLoop(HorovodGlobalState& state, MPIContext& ctx) {
     state.message_table = std::unique_ptr<MessageTable>(new MessageTable());
   }
 
+#if HAVE_MLSL
```

Comment:
Should we have an option to default to Gloo here, too?

Reply:
Because currently Gloo is slower than the other two. I thought we can ignore Gloo for now.

Reply:
Maybe we should default to Gloo if HOROVOD_CPU_OPERATIONS=GLOO was specified during installation?  And if HOROVOD_CPU_OPERATIONS=MPI was specified, then even if MLSL was present it won't be enabled by default?

Reply:
Fixed.